### PR TITLE
What we need to get PDF doc to build...

### PR DESCRIPTION
**Pull request by rocky**
In the last release, it was difficult enough to get things to build. Going over documentation suffered a little.
I am now starting to do this as part of releasing new docker images. (The last one was 4 months ago)

* Some references corrected.
* Remove duplicate sections (in both test and doc) in guide sections
* Fix a small markup typo in ComplexExpand formatting
* Summary text now appears in non-guide Chapters which have sections...
* Also start to go over are regularize summary text.
* Remove unused Builtin field "abstract".

### CODE REVIEW

File: mathics/builtin/datentime.py [hunk 1]
```diff
@@ -987,7 +986,7 @@ class DateStringFormat(Predefined):
 
     <dl>
       <dt>'$DateStringFormat'
-      <dd>gives the format used for dates generated by 'DateString'.
+      <dd>get the format used for dates generated by 'DateString'.
```
Comment (mmatera) [inline mathics/builtin/datentime.py:989]: I had the feeling that in the docstring we used the style "[Expression] does/returns/gives ..." using a third person in the simple present tense, while the `summary_text` was in the imperative form, wasn't it?

Comment (rocky) [reply mathics/builtin/datentime.py:989]: This should have just been on the summary text. I'll change this. 

In the summary text it is 3rd person active. "get" or "give" would work but for summary text "get" seems to be what is used most.

**Review by mmatera — APPROVED** [review]
LGTM

**Comment by rocky** [comment]
Should commit before I start adding more (of which I am sure there is a lot more to do).

**Pull request merged**